server/services/screenshot_match.py

```
@@ -0,0 +1,268 @@
+    unmatched: list[tuple[int, DetectedGarment]] = []
+    for idx, garment in enumerate(garments):
+        match = _best_match(garment, items)
+        results.append(ScreenshotItemResult(garment=garment, wardrobe_match=match))
+        if match is None and catalog is not None:
+            unmatched.append((idx, garment))
+
+    if unmatched and catalog is not None:
+        suggestions = await asyncio.gather(
+            *(_shop_for(g, catalog, affiliate_ok) for _, g in unmatched)
+        )
+        for (idx, _), suggestion in zip(unmatched, suggestions):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)